wandb/sdk/wandb_settings.py

```diff
@@ -1361,7 +1361,8 @@ def _infer_settings_from_environment(
         if os.path.exists("/usr/local/cuda/version.txt"):
             with open("/usr/local/cuda/version.txt") as f:
                 settings["_cuda"] = f.read().split(" ")[-1].strip()
-        settings["_args"] = sys.argv[1:]
+        if not self._jupyter:
```

Comment:
will this apply to colab/sagemaker as well?

Reply:
Yes, but I thought that self._jupyter was set in both of those cases as well (well for sagemaker notebooks)